src/cr/cube/crunch_cube.py

```diff
@@ -837,3 +838,35 @@ def index(self, axis, weighted=True):
             margin = margin[:, np.newaxis]
 
         return np.true_divide(props, margin)
+
+    @property
+    def z_scores(self):
+
+        if self.has_mr:
+            return self._mr_z_scores
+
+        counts = self.as_array()
+        total = self.margin()
+        p_col = self.margin(axis=0) / total
+        p_row = self.margin(axis=1) / total
+        expected_counts = chi2_contingency(counts)[3]
```

Comment:
would `scipy.stats.contingency.expected_freq` be clearer than ‘the fourth thing it spits out’?

Reply:
:+1: